ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -106,6 +106,13 @@ def _prepareCommand( self ):
       if not seHost:
         return S_ERROR( 'No seHost for %s' % elementName )
       elementName = seHost
+      
+    elif elementType == 'FTS':
+    	gocdbServiceType = "FTS"
+    	try:
+    		elementName	= getFTSServers("FTS3")[ 'Value' ][0]
+    	except:
+    		return S_ERROR( 'No FTS3 server specified in dirac.cfg (see Resources/FTSEndpoints)' )
```

Comment:
no, it would be 
FTSEndpoints
{
FTS3
{
  FTS3-RAL = https://blalbla
  FTS3-CERN = https://blibli
}
}

If you look at the getFTS3Servers method (which I still don't understan why you mangled the names), you will see it.